.github/workflows/ci_tests.yaml

```diff
@@ -135,7 +135,7 @@ jobs:
 
       # Run full tests including doctests on Wednesday
       - name: Run full tests
-        if: github.event_name == 'schedule' && github.event.schedule == '0 0 * * 3'
+        if: github.event_name == 'schedule' && endswith(github.event.schedule, '3')
```

Comment:
This should hopefully trigger the full tests on Wednesday, inspired by https://github.com/orgs/community/discussions/25662#discussioncomment-3248648.

However, I have a feeling that there will be two runs (tests and fulltests) because L133's `if: github.event.schedule != '0 0 * * 3'` isn't quite right...